### ISSUE TITLE
license Check dashboard

### ISSUE THREAD

**Issue by arzakhar** [porscheinformatik/sonarqube-licensecheck#287]
I use the following tool version
```
INFO: SonarScanner 4.6.2.2472
INFO: Java 11.0.11 AdoptOpenJDK (64-bit)
INFO: Linux 4.15.0-159-generic amd64
INFO: Analyzing on SonarQube server 8.9.1
```
licensecheck successfully finds license and uploads them into sonarqube instance. I see them in issue tab, but the problem with license Check dashboard, it shows nothing. Could you clarify what could be the problem
```
INFO: Dependency @angular-architects/module-federation uses a not allowed license MIT License 
INFO: Dependency @angular/animations uses a not allowed license MIT License 
INFO: Dependency @angular/cdk uses a not allowed license MIT License 
INFO: Dependency @angular/common uses a not allowed license MIT License 
INFO: Dependency @angular/compiler uses a not allowed license MIT License 
INFO: Dependency @angular/core uses a not allowed license MIT License 
INFO: Dependency @angular/forms uses a not allowed license MIT License 
INFO: Dependency @angular/material uses a not allowed license MIT License 
INFO: Dependency @angular/material-moment-adapter uses a not allowed license MIT License 
INFO: Dependency @angular/platform-browser uses a not allowed license MIT License 
INFO: Dependency @angular/platform-browser-dynamic uses a not allowed license MIT License 
INFO: Dependency @angular/router uses a not allowed license MIT License 
INFO: Dependency @fullhuman/postcss-purgecss uses a not allowed license MIT License 
INFO: Dependency @nrwl/angular uses a not allowed license MIT License 
INFO: Dependency @tailwindcss/postcss7-compat uses a not allowed license MIT License 
INFO: Dependency autoprefixer uses a not allowed license MIT License 
INFO: Dependency big-integer uses a not allowed license The Unlicense 
INFO: Dependency core-js uses a not allowed license MIT License 
INFO: Dependency cross-env uses a not allowed license MIT License 
INFO: Dependency cross-var uses a not allowed license MIT License 
INFO: Dependency file-saver uses a not allowed license MIT License 
INFO: Dependency focus-visible uses a not allowed license W3C Software Notice and License (2002-12-31) 
INFO: Dependency ical-generator uses a not allowed license MIT License 
INFO: Dependency lodash uses a not allowed license MIT License 
INFO: Dependency moment uses a not allowed license MIT License 
INFO: Dependency postcss uses a not allowed license MIT License 
INFO: Dependency postcss-focus-visible uses a not allowed license Creative Commons Zero v1.0 Universal 
INFO: Dependency rxjs uses a not allowed license Apache License 2.0 
INFO: Dependency sass-loader uses a not allowed license MIT License 
INFO: Dependency subsink uses a not allowed license MIT License 
INFO: Dependency tailwindcss uses a not allowed license MIT License 
INFO: Dependency url-search-params-polyfill uses a not allowed license MIT License 
INFO: Dependency zone.js uses a not allowed license MIT License 
INFO: Sensor License Check [licensecheck] (done) | time=132ms
INFO: Sensor OpenApi Linter Issues Loader Sensor [openapilinter]
INFO: Reflections took 41 ms to scan 1 urls, producing 2 keys and 2 values 
INFO: Sensor OpenApi Linter Issues Loader Sensor [openapilinter] (done) | time=134ms 
INFO: Analysis report uploaded in 72ms
INFO: ------------- Check Quality Gate status
INFO: Waiting for the analysis report to be processed (max 300s)
INFO: QUALITY GATE STATUS: PASSED
```

**Comment by ffuf-schweizer** [comment]
Hey @arzakhar , I noticed the same behavior. The dashboard does not take the branch into consideration when doing the api request. Therefore, no matter on which branch you are, it will always only show the dashboard information for the main branch. I also wonder if this is expected behavior or just not implemented yet.

**Comment by Janpopan** [comment]
See #119 

Maybe a solution can be found here

https://github.com/dependency-check/dependency-check-sonar-plugin/pull/274/files

**Comment by radenkoletic** [comment]
Any updates on this issue? I am using Sonarqube Developer edition `8.9 (build 43852)`. Noticed the same behavior as @arzakhar mentioned.

**Comment by Janpopan** [comment]
See also #119

And a Mergentheim reqeust to solve #325

**Comment by derkoe** [comment]
Fixed via #325
